src/components/Sidebar.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import "./Sidebar.css";
+import Home from "../components/icons/home";
+import Menu from "../components/icons/menu";
+import Down from "../components/icons/down";
+
+interface SidebarProps {
+  setTitle: (title: string) => void;
+}
+
+const Sidebar: React.FC<SidebarProps> = ({ setTitle }) => {
+  const [showDropdown, setShowDropdown] = useState<boolean>(false);
+
+  return (
+    <div className="sidebar">
+      <div className="menu-item" onClick={() => setTitle("Start")}>
+        <Home /> Start
+      </div>
+      <div className="menu-item" onClick={() => setShowDropdown(!showDropdown)}>
+                  <Menu />Machine Learning &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<Down/>
+       
+      </div>
+      {showDropdown && (
+        <div className="dropdown">
+          <div className="dropdown-item" onClick={() => setTitle("Example 1")}>
+            Example 1
```

Comment:
'Example 1', 'Example 2', 'Example 3' is a list of menu entries and it can change. That is why it is better to keep it in some constant and render it here dynamically. This way, if you wanted to add new entry, for example 'Example 4' you would just need to update constant holding it without changes to component. Please extract this into constant and render it dynamically.